yogstation/code/modules/clothing/head/misc.dm

```diff
@@ -24,4 +24,19 @@
 /obj/item/clothing/head/yogs/goatpelt/king/dropped(mob/living/carbon/human/user)
 	..()
 	if (user.head == src)
-		user.faction -= "goat"
\ No newline at end of file
+		user.faction -= "goat"
+
+/obj/item/clothing/head/helmet/juggernaut
+	name = "Juggernaut Helmet"
+	desc = "I...am...the...JUGGERNAUT!!!."
+	alternate_worn_icon = 'yogstation/icons/mob/head.dmi'
+	icon = 'yogstation/icons/obj/clothing/hats.dmi'
+	icon_state = "juggernauthelm"
+	item_state = "juggernauthelm"
+	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 90)
```

Comment:
I hadn't done the math yet but I think people might end up stacking armor in order to make themselfs meleeproof

Reply:
Eh not that big of the deal you get slowed down so that makes it a bit annoying to use plus you can still meme em using disablers if they start attacking the crew for some reason